solr/core/src/test/org/apache/solr/search/TestSolrQueryParser.java

```diff
@@ -1057,7 +1057,35 @@ public void testShingleQueries() throws Exception {
         , "/response/numFound==1"
     );
   }
-  
+
+
+  public void testSynonymQueryStyle() throws Exception {
+    ModifiableSolrParams edismaxParams = params("qf", "t_pick_best_foo");
+
+    QParser qParser = QParser.getParser("tabby", "edismax", req(edismaxParams));
```

Comment:
Why not the default/lucene query parser?  That's what TestSolrQueryParser tests.

Reply:
whoops, good catch